lib/pavilion/utils.py

```diff
@@ -1,16 +1,39 @@
 # This file contains assorted utility functions.
 
-from pathlib import Path
 import csv
+import datetime
+import itertools
 import json
 import os
 import re
+import shutil
 import subprocess
 import sys
 import textwrap
-import shutil
-import itertools
 from collections import defaultdict
+from pathlib import Path
+
+def get_relative_timestamp(base_dt):
+    """Print formatted time string based on the delta of time objects.
+    :param obj datetime object
+    :rtype string
+    """
+    now = datetime.datetime.now()
+    format_ = ['%Y', '%b', '%a', '%H:%M:%S'] # year, month, day, time
+
```

Comment:
for i in range(len(format_)):
  if now.strftime(format_[i]) != ....:
     return now.strftime(" ".join(format_[i:])